go-controller/pkg/ovn/master_test.go

```diff
@@ -875,6 +886,96 @@ var _ = ginkgo.Describe("Gateway Init Operations", func() {
 		wg = &sync.WaitGroup{}
 
 		libovsdbCleanup = nil
+
+		node1 = tNode{
+			Name:                 "node1",
+			NodeIP:               "1.2.3.4",
+			NodeLRPMAC:           "0a:58:0a:01:01:01",
+			LrpMAC:               "0a:58:64:40:00:02",
+			LrpIP:                "100.64.0.2",
+			LrpIPv6:              "fd98::2",
+			DrLrpIP:              "100.64.0.1",
+			PhysicalBridgeMAC:    "11:22:33:44:55:66",
+			SystemID:             "cb9ec8fa-b409-4ef3-9f42-d9283c47aac6",
+			NodeSubnet:           "10.1.1.0/24",
+			GWRouter:             types.GWRouterPrefix + "node1",
+			GatewayRouterIPMask:  "172.16.16.2/24",
+			GatewayRouterIP:      "172.16.16.2",
+			GatewayRouterNextHop: "172.16.16.1",
+			PhysicalBridgeName:   "br-eth0",
+			NodeHostAddress:      []string{"9.9.9.9"},
+			NodeGWIP:             "10.1.1.1/24",
+			NodeMgmtPortIP:       "10.1.1.2",
+			NodeMgmtPortMAC:      "0a:58:0a:01:01:02",
+			DnatSnatIP:           "169.254.0.1",
+		}
+
+		expectedClusterLBGroup := newLoadBalancerGroup()
+		expectedOVNClusterRouter := newOVNClusterRouter()
+		expectedNodeSwitch := node1.logicalSwitch(expectedClusterLBGroup.UUID)
+		expectedClusterRouterPortGroup := newRouterPortGroup()
+		expectedClusterPortGroup := newClusterPortGroup()
+
+		dbSetup = libovsdbtest.TestSetup{
+			NBData: []libovsdbtest.TestData{
+				newClusterJoinSwitch(),
+				expectedNodeSwitch,
+				expectedOVNClusterRouter,
+				expectedClusterRouterPortGroup,
+				expectedClusterPortGroup,
```

Comment:
nit: I think you could inline the creation of these structs instead of assigning them to variables, then using them, since you're not (afaict) using the variables anywhere else.

Reply:
@maiqueb changed them to inline except for 
```
expectedClusterLBGroup 
expectedNodeSwitch 
```
because I needed the UUID later. Is that okay?